util/fqdn_test.go

```diff
@@ -31,7 +31,7 @@ func TestIsFQDNQuestionMarkFQDN(t *testing.T) {
 }
 
 func TestIsFQDNQuestionMarkIncorrectPlaceFQDN(t *testing.T) {
-	domain := "?.?.abc.?com"
+	domain := "?.?.abc?.com"
```

Comment:
I have literally no idea why this test was all of a sudden breaking. Something changed in the underlying govalidator library that we rely on to check if something is an FQDN.